internal/cfgplugins/interface.go

```diff
@@ -33,28 +33,29 @@ const (
 	targetFrequencyToleranceMHz   = 100000
 )
 
-// opticalChannelComponentFromPort returns the name of the optical channel component for the given
+// OpticalChannelComponentFromPort returns the name of the optical channel component for the given
 // port.
-func opticalChannelComponentFromPort(t *testing.T, dut *ondatra.DUTDevice, p *ondatra.Port) string {
+func OpticalChannelComponentFromPort(t *testing.T, dut *ondatra.DUTDevice, p *ondatra.Port) string {
```

Comment:
Oh interesting....we actually just went through this in a different place for this PR:
https://github.com/openconfig/featureprofiles/pull/2891

https://github.com/hattikals/featureprofiles/blob/3a34ffd7466037956b25ddea4ffbe34aae09e7ee/internal/components/components.go#L203

I don't know why we have this same code in two places :/

We should probably merge this somehow and only have one. Maybe just to start in this PR, can you just copy the logic from the other file?

Reply:
I recommended this change offline.  Yes, the goal should be to migrate to using `internal/cfgplugins` for common configuration functions.  For ease of review, all the migration does not need to happen in this PR.

Reply:
Agree, we should definitely break that up in multiple PRs.

Reply:
Ack - will revert to the tests using components.OpticalChannelComponentFromPort approved as part of #2891